jetstream/analysis.py

```diff
@@ -184,7 +184,9 @@ def _calculate_metrics(
             dry_run_query(sql)
         else:
             self.logger.info(
-                "Executing query for %s (%s)", self.config.experiment.normandy_slug, period.value,
```

Comment:
mh, looks like the custom line-length config for black gets ignored here :|

Reply:
I think black's format for function calls changed.